packages/protocol/migrationsConfig.js

```diff
@@ -73,11 +73,16 @@ const defaultConfig = {
 
 const linkedLibraries = {
   LinkedList: ['AddressLinkedList', 'SortedLinkedList'],
-  SortedLinkedList: ['AddressSortedLinkedList', 'IntegerSortedLinkedList'],
+  SortedLinkedList: [
+    'AddressSortedLinkedList',
+    'IntegerSortedLinkedList',
+    'SortedLinkedListWithMedian',
+  ],
+  SortedLinkedListWithMedian: ['AddressSortedLinkedListWithMedian'],
   AddressLinkedList: ['Validators'],
   AddressSortedLinkedList: ['Validators'],
   IntegerSortedLinkedList: ['Governance', 'IntegerSortedLinkedListTest'],
-  SortedFractionMedianList: ['SortedOracles', 'SortedFractionMedianListTest'],
+  AddressSortedLinkedListWithMedian: ['SortedOracles', 'AddressSortedLinkedListWithMedianTest'],
```

Comment:
Should test contracts live in a separate migrations config?
On the flipside might be dangerous to separate test/prod migrations

Reply:
I think this is fine since we don't actually migrate the test contract in our migrations.